default-configs/xtensa-softmmu.mak

```diff
@@ -10,6 +10,6 @@ CONFIG_XTENSA_XTFPGA=y
 CONFIG_SSP=y
```

Comment:
the commit subject should be more specific. Like this: ```Fix sdma size in IO space```

Reply:
I wanted to integrate these changes into an older commit that had this subject. Should I create a new commit instead?

Reply:
yes, please. That older commit was already merged. So will not be able to integrate the changes in that older commit .